packages/synapse-react-client/src/synapse-queries/featureflags/useGetFeatureFlags.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import SynapseClient from '../../synapse-client'
+import { SynapseClientError, useSynapseContext } from '../../utils'
+import { FeatureFlags } from '@sage-bionetworks/synapse-types'
+
+export function useGetFeatureFlags(
+  options?: Partial<UseQueryOptions<FeatureFlags, SynapseClientError>>,
+) {
+  const { keyFactory } = useSynapseContext()
+  return useQuery<FeatureFlags, SynapseClientError>({
+    staleTime: Infinity,
+    ...options,
+    queryKey: keyFactory.getFeatureFlagQueryKey(),
+    queryFn: () => SynapseClient.getFeatureFlags(),
+  })
+}
```

Comment:
Could do something similar to SWC where you make a custom UseQuery hook with the signature `useGetFeatureFlag(String FeatureFlagEnum)`. I think it would be pretty simple, something like this:

```suggestion
}

export function useGetFeatureFlag(
  featureFlag: FeatureFlagEnum
  options?: Partial<UseQueryOptions<boolean, SynapseClientError>>,
) {
  return useGetFeatureFlags({
    select: featureFlags => featureFlags[featureFlag]
  })
}
```